cvat-ui/src/base.scss

```diff
@@ -13,6 +13,8 @@ $layout-lg-grid-color: rgba(0, 0, 0, 0.15);
 
 $header-color: #d8d8d8;
 $text-color: #303030;
+$text-color-2: white;
```

Comment:
Need to update header

Reply:
I belive its ok. When we gonna implement new ui for user homepage and something like that, probably we will use such declarations

Reply:
I would suggest do not try to guess what we will do in the future..

Reply:
Ok. I've moved those declarations in corresponding file.